package.json

```diff
@@ -38,6 +38,7 @@
     "build": "npm run provision && npm test",
     "clean": "git clean -df",
     "rebuild": "npm run clean && npm run build",
+    "update": "rm -rf node_modules && rm npm-shrinkwrap.json && npm run provision && npm shrinkwrap && bin/cfupdate",
```

Comment:
Can we extract this line as a script in bin? Perhaps merge it with cfupdate one and rename everything to module-update?